contracts-js/ValidateSPV.js

```diff
@@ -0,0 +1,315 @@
+// pragma solidity ^0.5.10;
+
+// /** @title ValidateSPV*/
+// /** @author Summa (https://summa.one) */
+
+// import {BytesLib} from "./BytesLib.sol";
+// import {SafeMath} from "./SafeMath.sol";
+// import {BTCUtils} from "./BTCUtils.sol";
+
+
+// library ValidateSPV {
+const ValidateSPV = {
+
+//     using BTCUtils for bytes;
+//     using BTCUtils for uint256;
+//     using BytesLib for bytes;
+//     using SafeMath for uint256;
+
+//     enum InputTypes { NONE, LEGACY, COMPATIBILITY, WITNESS }
+//     enum OutputTypes { NONE, WPKH, WSH, OP_RETURN, PKH, SH, NONSTANDARD }
+
+  ERR_BAD_LENGTH: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff,
+  ERR_INVALID_CHAIN: 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe,
+  ERR_LOW_WORK: 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffd,
+
+//     function getErrBadLength() internal pure returns (uint256) {
+//         return ERR_BAD_LENGTH;
+//     }
+
+  getErrBadLength: () => {
```

Comment:
Can remove these getters. They're required in Solidity, but not so much in JS

Reply:
Done